src/runtime/tests/schema-test.ts

```diff
@@ -635,4 +636,15 @@ describe('schema', () => {
     const schema2 = getSchemaFromManifest(manifest, 'schema2');
     assert.isFalse(schema1.isAtleastAsSpecificAs(schema2));
   });
+  it.only('tests warning when refinement specificity is unknown', async () => {
```

Comment:
Please remove the `.only` otherwise all the tests will be disabled :)

I might see if I can get a lint check in for it.